### PR TITLE
Redesign welcome modals

### PR DESCRIPTION
**Pull request by mazameli**
Implements https://github.com/metabase/metabase/issues/5139

I could use some help from @kdoh or @attekei to bring this over the finish line:
* I'm not sure how to trigger the welcome modals via the JS console or whatevs (since they only show up the first time a new user logs in), which makes this tough to test
* Not sure what CSS trickery I'd need to make the illustrations flush with the frame (also not even sure this is a problem because I can't check my work due to the above)
* Not sure how best to implement the progress dots

### CODE REVIEW

**Comment by mazameli** [comment]
Just spun this up to check in on it:

1. The second modal is a different height than the other two; would be cool if there were all consistent so the `Next` button doesn't move around
2. The little dots need `cursor: pointer`
3. Seems like images are broken? Screenshot:

screen shot 2017-05-23 at 12 08 34 pm

**Comment by kdoh** [comment]
@mazameli Yup, still WIP.

File: frontend/src/metabase/home/components/NewUserOnboardingModal.jsx [hunk 1]
```diff
@@ -1,97 +1,115 @@
...
+                                currentStep={step}
+                                steps={STEPS}
+                                goToStep={step => this.setState({ step })}
+                            />
                         </div>
+                        <a
+                            className="link flex-align-right text-bold absolute right"
+                            onClick={() => (this.nextStep())}
+                        >
+                            Next
```
Comment (attekei) [inline frontend/src/metabase/home/components/NewUserOnboardingModal.jsx]: Should we have a different link text for the last step that indicates that the welcome tutorial will end?

File: frontend/src/metabase/home/components/NewUserOnboardingModal.jsx [hunk 2]
```diff
@@ -1,97 +1,115 @@
...
+                        </a>
                     </div>
-                }
+                </div>
             </div>
         );
     }
 }
+
+const OnboardingImages = ({ currentStep }, { currentStep: number }) =>
```
Comment (attekei) [inline frontend/src/metabase/home/components/NewUserOnboardingModal.jsx]: I think that `currentStep` isn't a number =)

**Review by attekei — APPROVED** [review]
Two notes but generally looks good (and opens properly after the setup wizard).

File: frontend/src/metabase/home/components/NewUserOnboardingModal.jsx (hunk 2 above)
Comment (kdoh) [inline frontend/src/metabase/home/components/NewUserOnboardingModal.jsx]: Whoops, yeah I changed it. I wonder why flow didn't catch that.

File: frontend/src/metabase/home/components/NewUserOnboardingModal.jsx (hunk 1 above)
Comment (kdoh) [inline frontend/src/metabase/home/components/NewUserOnboardingModal.jsx]: Derp, yeah my bad.

**Pull request merged**